src/query/api/v1/handler/prometheus/native/read.go

```diff
@@ -34,8 +34,8 @@ import (
 	"github.com/m3db/m3/src/query/ts"
 	"github.com/m3db/m3/src/query/util/httperrors"
 	"github.com/m3db/m3/src/query/util/logging"
-	opentracingutil "github.com/m3db/m3/src/query/util/opentracing"
 	xhttp "github.com/m3db/m3/src/x/net/http"
+	xopentracing "github.com/m3db/m3/src/x/opentracing"
```

Comment:
nit: can probably drop the `xopentracing` alias and leave it as `opentracing`

Reply:
Wanted to distinguish from the opentracing packages below